app/lib/pantry_trak/client.rb

```diff
@@ -69,5 +71,15 @@ def create_reservation_path
         'api/create_freshtrak_reservation_beta.php'
       end
     end
+
+    def handle_unsucessful_response(response)
+      # TODO: In order to rescue the very specific exceptions,
+      # refs# https://github.com/lostisland/faraday/blob/master/lib/faraday/error.rb
+      # Trying to handle exception and print on logs
+      Jets.logger.info '_' * 10
+      Jets.logger.error "Problem accessing PantryTrak web service,
+        Message: #{response.message}"
```

Comment:
Is there any other information that would be useful here? For example, the http status code from the failed api call. Not sure if that info is wrapped in the message.

Reply:
**response message looks like this :**

Problem accessing PantryTrak web service,
        Message: 783: unexpected token at '<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /core/admin/freshtrak_sync/api/create_freshtrak_user_betsdgfha.php was not found on this server.</p>
<hr>
<address>Apache/2.2.15 (CentOS) Server at secure.pantrytrak.com Port 80</address>
</body></html>